docs/rollover.md

```diff
@@ -0,0 +1,70 @@
+# Rollover
+
+Each year we close the current cycles courses and open the new cycle in a
+process we call 'Rollover'.
+
+This involves copying existing providers and courses to new records to allow the
+providers to update any details and then switching the API over to a new
```

Comment:
Should we mention why we copy? i.e. the current cycle is still open for applications so we don't want to make any changes to those courses..

Reply:
good idea, will add explanation